packages/sdk/src/resolve/resolve-cadence.js

```diff
@@ -1,20 +1,42 @@
 import {isTransaction, isScript, get} from "../interaction/interaction.js"
 import {invariant} from "@onflow/util-invariant"
 import {config} from "@onflow/config"
+import * as logger from "@onflow/util-logger"
 
 const isFn = v => typeof v === "function"
 const isString = v => typeof v === "string"
 
+const oldIdentifierPatternFn = () => /\b(0x\w+)\b/g
+function isOldIdentifierSyntax(cadence) {
+  return oldIdentifierPatternFn().test(cadence)
+}
+
+const newIdentifierPatternFn = () => /import\s+"(\w+)"/g
```

Comment:
What about the case here where someone writes `import "Foo" from "Alice"`. How do we want to handle that? Is that supported? If it isn't, should that be handled gracefully cc @sideninja

Reply:
I don't believe that format is supported.

Reply:
Let's confirm with @sideninja

Reply:
Yeah that format is no longer supported.

Reply:
thanks @sideninja sounds good